api/database/models.py

```diff
@@ -55,7 +55,9 @@ class Applicant(db.Model):
     # The name of the column is the name you assign it to
     id = Column(Integer, primary_key=True)
     email = Column(String(100), unique=True, nullable=False)
-    first_name = Column(db.String(80))
+    username = Column(String(100), default='Anonymous Giraffe')
+    bio = Column(String(100))
```

Comment:
sounds like @HireUp-Turing/front-end is wanting this to be 250. I'll take care of it on a new ticket in a little bit